### DISCUSSION TITLE
Auto-generate Reverse Alias for Trusted Emails

### DISCUSSION THREAD

**Discussion by ddulic** [simple-login/app#913] [Feature Requests]
Hello,

Have been a Premium user for a few months now and the main feature request I have is to have the ability to bypass or auto-generate a reverse alias in the background for certain trusted emails.

I am mainly noticing the issue when having an email chain with a third party that includes my partner, having effectively to generate a reverse alias for 2 emails.

Thanks.

**Comment by nguyenkims** [comment]
@ddulic I'm not sure to understand the feature ... What do you mean by "auto-generate a reverse alias in the background for certain trusted emails"? By default SimpleLogin automatically creates a reverse alias for all people who participate in an email conversation.

**Reply by ddulic** [reply]
The feature is when I am sending the email, or attempting to add my “trusted contact” as a CC to the email. I have to go to SimpleLogin, go under the alias and generate a unique reverse alias for that contact. If I don’t I receive the following email from SimpleLogin:

```
Hi

Your email sent to user@email.com from me@me.com (Email Title) contains an address which isn't a reverse-alias in the To: or Cc: field.

This will reveal your real mailbox address.

Please create a reverse alias for each recipient to make sure your mailbox stays hidden.

More info about reverse-alias can be found on https://simplelogin.io/docs/getting-started/reverse-alias/
and how to send an email from your alias on https://simplelogin.io/docs/getting-started/send-email/


Best,
SimpleLogin team.
```

**Reply by nguyenkims** [reply]
Oh I see, in this case maybe you can put the trusted contact in the BCC instead?

**Reply by ddulic** [reply]
True, but there are some emails where I want them to be clearly included.

**Reply by nguyenkims** [reply]
I'm curious: why don't you want to create a reverse alias for your trusted contact and use it in CC?

**Reply by ddulic** [reply]
I can, and I do, but it is an additional step, I have to do it under every single reverse alias...

Not to mention that if I want to add my partner I have to go to SimpleLogin every time to check which one to use as I currently have over 30 of them. The naming doesn't indicate under which Alias my partner belongs in. Yes, I could technically rename the contacts, but this is again more work...

Like I said it would be good if it is possible for SimpleLogin to interject the email, see the "trusted contact" and auto-generate a reverse alias under the correct Alias.

**Reply by nguyenkims** [reply]
@ddulic I see, thanks for the info. You can always add your partner address in CC and the email will be sent to them that includes the reverse alias. SimpleLogin can't intervene here as your email service will send the email directly to your partner mailbox, without going through SimpleLogin. The "Please create a reverse alias for each recipient" email you receive is rather for information only to prevent future mistake but if this is intentional, you can just ignore this email.

**Reply by ddulic** [reply]
Thanks for the info.

> The "Please create a reverse alias for each recipient" email you receive is rather for information only to prevent future mistake but if this is intentional, you can just ignore this email.

So, only my partner will see this email, it won't be leaked to the primary recipient? I believe the email I got seemed to suggest my email address would be leaked to all.

**Reply by nguyenkims** [reply]
@ddulic Yes this email won't be forwarded to your contact.

**Reply by ddulic** [reply]
Thanks for the info @nguyenkims 

I would like to see a change in wording in the email to make it more clear that it is for informational purposes only.

You are welcome to close this discussion.

**Reply by nguyenkims** [reply]
@ddulic we have updated the email wording in https://github.com/simple-login/app/blob/345b3ea4f01234d936aeb7065f6e53dada3391af/templates/emails/transactional/non-reverse-alias-reply-phase.txt.jinja2#L11

**Reply by tangiblehologram** [reply]
It would be very nice to be able to automatically reverse alias conversations I initiate.  I'll bet this could be done for Proton Mail.
